Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -12,7 +12,7 @@
                         unicode_literals)
 
 __author__ = ('Manodeep Sinha')
-__all__ = ('DDtheta_mocks',)
+__all__ = ('DDtheta_mocks', 'find_fastest_DDtheta_mocks_bin_refs_bin_refs')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Undefined variable name 'find_fastest_DDtheta_mocks_bin_refs_bin_refs' in __all__](https://app.codacy.com/manual/manodeep/Corrfunc/pullRequest?prid=5097606)

Reply:
Issue resolved to correct typo in L15.